pandas/tests/frame/test_constructors.py

```diff
@@ -1373,7 +1360,7 @@ def test_constructor_list_of_dict_order(self):
             }
         )
         result = DataFrame(data)
-        tm.assert_frame_equal(result, expected, check_like=not PY36)
+        tm.assert_frame_equal(result, expected, check_like=False)
```

Comment:
`check_like` defaults to `False` so it doesn't need to be specified here; same with the other instances of this in the PR.

https://github.com/pandas-dev/pandas/blob/6dbd2b1142a1fa1be33f031d319ee2ef7a49faef/pandas/util/testing.py#L1236